core/templates/dev/head/css/oppia.css

```diff
@@ -1195,11 +1195,34 @@ pre.oppia-pre-wrapped-text {
   border-radius: 4px;
 }
 
+.oppia-editable-section:hover pre {
+	background-color: inherit;
+}
+
+.oppia-editable-section:hover oppia-noninteractive-link,
+.oppia-editable-section:hover oppia-noninteractive-video,
+.oppia-editable-section:hover oppia-noninteractive-tabs {
+	pointer-events: none;
+	cursor: pointer;
+}
+
+.oppia-editable-section:hover oppia-noninteractive-tabs li.active a {
+	background-color: inherit;
+	-webkit-transition : all 200ms;
+	transition: all 200ms;
+}
+
 .oppia-editable-section:hover .oppia-interaction-preview {
   background: rgba(5, 140, 166, 0.5);
   border-radius: 4px;
 }
 
+.oppia-editable-section:hover oppia-noninteractive-image img {
+    opacity: 0.7;
```

Comment:
Well it's same CSS class i added previously to Images for graymasking.
Adding it to other RTE components not required. Since it does not do any new thing.
Except image we only have to graymask pre and video components. 
Since pre has inherit background we won't need transition.
And changing opacity in video does nothing.
Thats why we need transition and opacity only in Image component.